lib/Doctrine/Common/Cache/MemcachedCache.php

```diff
@@ -92,6 +92,16 @@ protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
     /**
      * {@inheritdoc}
      */
+    protected function doDeleteMultiple(array $keys)
+    {
+        $stat = $this->memcached->deleteMulti($keys);
+        return count(array_unique($stat) > 1)
+            || end($stat) === true;
```

Comment:
@flack yeah, we can't guarantee consistency across all drivers (unless we check for existence and then delete), but when the driver may fail, then we should at least test that scenario. Mocking out memcached is a way to achieve this in this scenario, for example (doesn't need to be an actual functional test)